test/app-tap/gh-1700-abort-recording-on-fiber-switch.test.lua

```diff
@@ -41,10 +41,9 @@ if #arg == 0 then
                           '<LUABIN> 2>&1 <SCRIPT>', '%<(%w+)>', vars)
 
   for _, ch in pairs(checks) do
-    local res
     local proc = io.popen((cmd .. (' %s'):rep(#ch.arg)):format(unpack(ch.arg)))
-    for s in proc:lines() do res = s end
-    assert(res, 'proc:lines failed')
+    local res = proc:read()
+    assert(res, 'proc:read failed')
```

Comment:
Why do you need to update the test?

Reply:
Check out this diff:
https://github.com/tarantool/tarantool/pull/6927/files#diff-0724515e6d62d9c2b0fdf3185945a73b24d5fe41ec8941b91e95b5f918635b0aL594-L596.

This test doesn't expect the backtrace to be printed to `STDOUT`.

Reply:
I don't understand. This patch is supposed to just refactor the code so it should have no visible user changes. In particular, all tests should continue to work without any modifications.

Reply:
Fixed: the backtrace in `tarantool_panic_handler` is printed to STDERR, as it was originally — this test works again.